stim_selective_stop_signal/experiment.js

```diff
@@ -646,6 +703,12 @@ var practice_node = {
         practice_feedback_text +=
           '</p><p class = block-text>Your accuracy is too low. Remember, the correct keys are as follows: ' + prompt_text
       }
+      
+      if (ignoreRespond_percent >= .4){
```

Comment:
Don't you want this to be <=.4? You are tracking how many times they respond to the ignore stimulus, which they should be doing.  I also think you want "<=.6" to keep in line with your "getSelectiveFeedback" function. You may want to define this as an "ignore_thresh" similar to how we define the other thresholds.